server/model/monitor.js

```diff
@@ -971,7 +982,13 @@ class Monitor extends BeanModel {
             } else if (bean.status === MAINTENANCE) {
                 log.warn("monitor", `Monitor #${this.id} '${this.name}': Under Maintenance | Type: ${this.type}`);
             } else {
-                log.warn("monitor", `Monitor #${this.id} '${this.name}': Failing: ${bean.msg} | Interval: ${beatInterval} seconds | Type: ${this.type} | Down Count: ${bean.downCount} | Resend Interval: ${this.resendInterval}`);
+                let groupText = "";
+                if (this.type !== "group" && this.parent !== "") {
+                    let parentName = await this.getParentName();
+                    groupText = ` under group '${parentName}'`;
```

Comment:
Re https://github.com/louislam/uptime-kuma/pull/4371/files#r1452660704 

(I can't add the second line to the suggestion)

I did forget to add the second line to the code suggestion.
I don't think the comparison with `getPathName` is fair as said function does do a lot more.

```suggestion
                    let parent = await Monitor.getParent(this.id);
                    groupText = ` under group '${parent.name}'`;
```